Apps/phradar/radar.json

```diff
@@ -1,876 +1,871 @@
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "Status",
+                    "column_order": 3,
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "column_name": "Message",
+                    "column_order": 4,
+                    "example_values": [
+                        "Successfully added note"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "custom",
+                "view": "radar_view.display_view",
+                "title": "Privacy Incident Note",
+                "menu_name": "Radar Privacy Incident Note"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get notes",
+            "description": "Get all notes for a Radar privacy incident",
+            "verbose": "This action gets all notes for a given Radar incident using the incident ID supplied in the action parameters. Only notes that were created with the 'create note' action will be fetched from Radar.",
+            "type": "investigate",
+            "identifier": "get_notes",
+            "read_only": true,
+            "parameters": {
+                "incident_id": {
+                    "description": "Radar incident ID",
+                    "data_type": "numeric",
+                    "required": true,
+                    "order": 0,
+                    "contains": [
+                        "radar incident id"
+                    ],
+                    "primary": true
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.incident_id",
+                    "data_type": "numeric",
+                    "column_name": "Incident ID",
+                    "column_order": 1,
+                    "contains": [
+                        "radar incident id"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.*.id",
```

Comment:
If we can avoid `*.*.` then let's make changes in the connector code

Reply:
Fixed

Reply:
@alexa-phantom , @dhwanis-crest I would like to point out that this actually did break the view display for the action. Our team has finally gotten some bandwidth to test the changes that were introduced here.

Reply:
Just stating this as a matter of fact, not to point fingers at this point in time. I understand there was a request for me to review these changes before they were merged into the next release branch, and I was somewhat late on getting around to review. However I would be interested to understand the decision to merge and release these changes without the app development team's approval. I understand that your team cannot manually test the changes in this PR, as there is a lack of access to our software, but it seems problematic to make updates and then release them without more assurance.

Reply:
Hi @pauwlsky , I understand your point. 
As a standard practice, we don't use *.* in the data path because that leads to a wrong number when we need to fetch the size of the result set using Phantom API: For example, summary['total_incidents'] = action_result.get_data_size()
To make sure about this @alexa-phantom suggested avoiding usage of *.* in the data path. 
I agreed that we should wait until you confirm that change. But many times we were facing a long-delayed from the contributor and it took a very long time to release the app on the portal. I will make sure that if any such changes we required as a part of standard practice then we will inform you and wait for the confirmation.
To resolve this, I would request you to update the custom-output view and resubmit the PR, we will work on that and release the app with the changes on the portal.

Reply:
Hi @pratikp-crest , thanks for your response. Your explanation about why `*.*.` is problematic makes sense, I'll keep it in mind for future work. 

I can understand the problems long delays waiting for contributer input can cause, I will make sure to get to requests for review earlier next time, and at least flag changes that need to go through QA before they can be released. It would be much appreciated if your team can wait to hear word from our team before releasing. We have tight resourcing on our end, and getting around to fixing issues can take quite a while, so it's important to us that we are sure the release is tested if changes like this are made. I will try to hop on this sooner that later though, and hopefully will have resubmitted a PR for v1.0.1 within the next couple weeks.